src/Utilities/ExcelImporterUtils.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace ExcelMapper.Utilities
+{
+    public static class ExcelImporterUtils
+    {
+#if NETSTANDARD2_0
+        public static List<ExcelClassMap> RegisterMapperClassesByNamespace(this ExcelImporter importer,
+            string namespacestr)
+        {
+            var classes = Assembly.GetExecutingAssembly()
+                .GetTypes().AsParallel()
+                .Where(t => t.IsClass && t.Namespace == namespacestr);
```

Comment:
Why not just do `t.IsAssignableFrom(typeof(ExcelClassMap)) && t.Namespace == namespace`